presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -286,33 +308,87 @@ private TableHandle getTableHandle(ShowStats node, QualifiedName table)
             return columnNames.stream().map(QueryUtil::unaliasedName).collect(toImmutableList());
         }
 
-        static List<String> buildColumnsNames(List<String> statisticsNames)
+        static List<String> buildStatsColumnsNames(List<String> statisticsNames)
         {
             ImmutableList.Builder<String> columnNamesBuilder = ImmutableList.builder();
-            columnNamesBuilder.add("column_name");
-            columnNamesBuilder.addAll(statisticsNames);
+            columnNamesBuilder.add(COLUMN_NAME_COLUMN);
+            columnNamesBuilder.addAll(sortedCopyOf(
```

Comment:
Is sorting important here? I know we want to be deterministic, but weren't we already?